lib/cli.py

```diff
@@ -82,6 +95,12 @@ def write_alignments(self):
         fn = os.path.join(str(self.arguments.input_dir), "alignments.{}".format(self.serializer.ext))
         if self.arguments.alignments_path is not None:
             fn = self.arguments.alignments_path
+        print("Alignments filepath: %s" % fn)
+        if os.path.exists(fn):
+            with open(fn, self.serializer.roptions) as inf:
+                data = self.serializer.unmarshal(inf.read())
+                for k, v in data.items():
+                    self.faces_detected[k] = v
```

Comment:
Thanks for your work :) I had a chance to test out the PR, this is what I found:

If I don't switch on the `--skip-existing` flag, and there's an old alignments file, it still gets merged with the new one. I suggest not merging with the old alignments file if `--skip-existing` isn't enabled, but if you wanna don't wanna go down that path and instead want to always merge there's a problem here:

The order you have it is:
1. Extract alignments into `self.faces_detected`
2. Use previous alignments file to overwrite any entries already in `self.faces_detected`
3. Write out merged alignments

I believe it should be the other way around:
1. Use previous alignments file to fill `self.faces_detected` during initialization
2. Extract alignments into `self.faces_detected`, overwriting any old alignments with the new extracted ones.
3. Write out merged alignments

This will ensure that the new alignments are kept.

Reply:
Totally missed deactivating the alignments merging when not using skip-existing. Fixing asap.